website/docs/cloud-docs/api-docs/policy-checks.mdx

```diff
@@ -357,9 +363,15 @@ This endpoint supports pagination [with standard URL query parameters](/cloud-do
 |----------------------------------- | ----------------------------------------------------------------------- |
 | `page[number]`                     | **Optional.** If omitted, the endpoint returns the first page.          |
 | `page[size]`                       | **Optional.** If omitted, the endpoint returns 20 agent pools per page. |
+| `filter[n][status]`                | **Optional.** If omitted, the endpoint returns all policies regardless of status. Must be one of: passed, failed, errored. |
+| `filter[n][enforcementLevel]`      | **Optional.** Only used if paired with a non-errored status filter. Must be one of: advisory, mandatory. |
```

Comment:
Should these be code font instead? I just want to set them apart from the typical text a bit.

```suggestion
| `filter[n][status]`                | **Optional.** If omitted, the endpoint returns all policies regardless of status. Must be either "passed", "failed", or "errored". |
| `filter[n][enforcementLevel]`      | **Optional.** Only used if paired with a non-errored status filter. Must be either "advisory" or "mandatory." |
```

Reply:
I think in quotes as suggested is good, thank you.